src/components/calcite-scrim/calcite-scrim.scss

```diff
@@ -12,30 +10,21 @@
 
 @keyframes calcite-scrim-fade-in {
   0% {
-    opacity: 0;
+    @apply bg-opacity-0;
   }
   100% {
-    opacity: 1;
+    @apply text-opacity-100;
   }
 }
 
 .scrim {
-  align-items: center;
+  @apply flex content-center items-center absolute bottom-0 left-0 right-0 top-0 select-none;
   animation: calcite-scrim-fade-in 250ms ease-in-out;
   background-color: var(--calcite-scrim-background);
```

Comment:
Question for others on this pattern.. is any objection to preferring `:host([theme="dark"]) &` in order to remove some indirection?